nautobot/extras/templates/extras/admin/plugins_list.html

```diff
@@ -1,4 +1,4 @@
-{% extends "admin/base_site.html" %}
```

Comment:
Note that this file (I mean `admin/plugins_list.html`) is being deleted in #935.